content/flyers.md

```diff
@@ -0,0 +1,13 @@
+type: page
+title: Flyers
+slug: flyers
+---
+
+Here are some flyers to promote the conference.
+Please feel free to share widely.
+
+* [Open Day](https://raw.githubusercontent.com/PyconUK/2016.pyconuk.org/master/resources/flyers/PyCon UK 2016 Open Day.pdf)
```

Comment:
I couldn't work out how to get wok to do the right thing with PDFs last night, but it was obvious when I looked at it again just now, so this should be fixed.